code/game/gamemodes/dynamic/dynamic_rulesets.dm

```diff
@@ -202,6 +203,13 @@
 				round = list(round)
 			if(name in round)
 				adjustment += repeated_mode_adjust[rounds_ago]
+				occurances += 1
+		if(occurances >= 2)
```

Comment:
there should be a config for max occurances

Reply:
I second this. 
Some things should be allowed more triggers than others.